d-sgov/d-sgov-vfr-1.8.0-diagram.ttl

```diff
@@ -174,7 +180,7 @@ d-sgov-vfr-1.8.0:diagram
                 "d-sgov-vfr-1.8.0-pojem" ;
         <http://purl.org/vocab/vann/preferredNamespaceUri>
                 "http://slovník.gov.cz/datový/vfr-1.8.0/pojem/" ;
-        owl:imports     d-sgov-vfr-1.8.0:glosář , <http://slovník.gov.cz/legislativní/sbírka/111/2009/diagram> ;
+        owl:imports     <http://slovník.gov.cz/dokumentový/ofn-adresy-1.0.1/diagram> , d-sgov-vfr-1.8.0:glosář , <http://slovník.gov.cz/legislativní/sbírka/111/2009/diagram> ;
```

Comment:
proč existuje slovník pro diagram?

Reply:
To není slovník, ale součást slovníku (glosář + model + 0..* diagramů). Integroval to Miro. Mělo yb to sloužit k tomu, že vidíš obrázky, ve kterých se vyskytujív termitu oanotované pojmy.